store/tikv/coprocessor.go

```diff
@@ -90,6 +90,7 @@ type copTask struct {
 	respChan  chan *copResponse
 	storeAddr string
 	cmdType   tikvrpc.CmdType
+	storeType StoreType
```

Comment:
seems this value is not initialized?

Reply:
Not initialize in this PR. It is equal to zero or `TiKV` in other words.